coc/ext/fullwarapi/__init__.py

```diff
@@ -0,0 +1,221 @@
+"""An extension that helps interact with the FullWar API."""
+
+import asyncio
+import base64
+import logging
+import json
+
+from collections import namedtuple
+from datetime import datetime
+from typing import List
+
+import aiohttp
+
+from ...http import json_or_text
+from ...utils import correct_tag
+from ...wars import ClanWar
+
+LOG = logging.getLogger(__name__)
+
+AccessToken = namedtuple("AccessToken", ["token", "expires_at"])
+
+
+def extract_expiry_from_jwt_token(token):
```

Comment:
Proto should be

```py
def extract_expiry_from_jwt_token(Union[str, bytes]) -> Optional[datetime]
```

Reply:
Looks like the fullwarapi is the only class that uses this function. This function should be made into a static function of the class.